typify-impl/src/lib.rs

```diff
@@ -94,13 +101,15 @@ pub struct TypeEnum<'a> {
     details: &'a type_entry::TypeEntryEnum,
 }
 
+#[allow(missing_docs)]
 /// Enum variant details.
 pub enum TypeEnumVariant<'a> {
     Simple,
     Tuple(Vec<TypeId>),
     Struct(Vec<(&'a str, TypeId)>),
```

Comment:
```suggestion
    /// Struct-type variant with named properties and types.
    Struct(Vec<(&'a str, TypeId)>),
```

Ugh... makes me realize that there's no way to get at the description for each variant.